soc/arm/st_stm32/stm32l1/Kconfig.series

```diff
@@ -0,0 +1,18 @@
+# Kconfig - ST Microelectronics STM32L1 MCU series
+#
+# Copyright (c) 2019 Linaro Ltd.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_SERIES_STM32L1X
+	bool "STM32L1x Series MCU"
+	select CPU_CORTEX_M3
+	select CPU_CORTEX_M_HAS_VTOR
```

Comment:
You don't need to select CPU_CORTEX_M_HAS_VTOR here because it's already selected by CPU_CORTEX_M3.

Reply:
Ack